react/src/lib/EventOverlay/index.js

```diff
@@ -607,10 +607,10 @@ class EventOverlay extends React.Component {
           ) {
             targetNode.style.bottom = `${documentScrollTop + windowBottom - documentBottom}px`;
           }
-          if (elementDims.right > documentRight || this.elementWidth > documentRight) {
+          if (elementDims.right >= documentRight || this.elementWidth > documentRight) {
             targetNode.style.right = '0px';
             if (this.elementWidth < documentRight) {
-              targetNode.style.left = `${documentRight - this.elementWidth}px`;
+              targetNode.style.left = 'inherit';
```

Comment:
The elementWidth is unreliable at this point. It has already been placed and if it is constrained by the right hand edge the box is likely too narrow (e.g. the vertical exit tooltip). Here I unset the left css rule so the tooltip can be placed naturally